lib/rex/post/meterpreter/extensions/stdapi/railgun/util.rb

```diff
@@ -382,43 +402,27 @@ def read_wstring(pointer, length = nil)
   # InitializeUnicodeStr(&uStr, sL"string");
   #
   def alloc_and_write_wstring(value)
```

Comment:
Might be worth a quick sanity check to see if these mixins/modules work still and/or need to be updated:

https://github.com/rapid7/metasploit-framework/blob/816048b9f564a56ecef5dd00e1622a62488a9593/lib/msf/core/post/windows/accounts.rb#L463-L481

https://github.com/rapid7/metasploit-framework/blob/816048b9f564a56ecef5dd00e1622a62488a9593/modules/exploits/windows/local/cve_2022_21999_spoolfool_privesc.rb#L315-L319

Reply:
Fixed in 5b5d5ade408e4e5cdc61979f49621a6731020e72